app/form/v1_0/steps/thinking-behaviours-attitudes.ts

```diff
@@ -29,43 +29,49 @@ const sectionConfig: SectionConfig = {
         thinkingBehavioursFields.thinkingBehavioursAttitudesProblemSolving,
         thinkingBehavioursFields.thinkingBehavioursAttitudesPeoplesViews,
         thinkingBehavioursFields.thinkingBehavioursAttitudesManipulativePredatoryBehaviour,
-        thinkingBehavioursFields.thinkingBehavioursAttitudesRiskSexualHarm,
+        thinkingBehavioursFields.thinkingBehavioursAttitudesTemperManagement,
+        thinkingBehavioursFields.thinkingBehavioursAttitudesViolenceControllingBehaviour,
+        thinkingBehavioursFields.thinkingBehavioursAttitudesImpulsiveBehaviour,
+        thinkingBehavioursFields.thinkingBehavioursAttitudesPositiveAttitude,
+        thinkingBehavioursFields.thinkingBehavioursAttitudesHostileOrientation,
+        thinkingBehavioursFields.thinkingBehavioursAttitudesSupervision,
+        thinkingBehavioursFields.thinkingBehavioursAttitudesCriminalBehaviour,
+        thinkingBehavioursFields.wantToMakeChanges(),
         thinkingBehavioursFields.isUserSubmitted(stepUrls.thinkingBehavioursAttitudes),
         thinkingBehavioursFields.sectionComplete(),
       ].flat(),
       navigationOrder: 8,
-      next: [
-        nextWhen(thinkingBehavioursFields.thinkingBehavioursAttitudesRiskSexualHarm, 'YES', stepUrls.sexualOffending),
-        nextWhen(thinkingBehavioursFields.thinkingBehavioursAttitudesRiskSexualHarm, 'NO', stepUrls.thinkingBehaviours),
-      ],
+      next: stepUrls.riskOfSexualHarm,
       sectionProgressRules: [setFieldToIncomplete(section.sectionCompleteField)],
     },
     {
-      url: stepUrls.sexualOffending,
+      url: stepUrls.riskOfSexualHarm,
       pageTitle: 'Risk of sexual harm',
       pageSubHeading: section.title,
       fields: [
-        thinkingBehavioursFields.thinkingBehavioursAttitudesSexualPreoccupation,
-        thinkingBehavioursFields.thinkingBehavioursAttitudesOffenceRelatedSexualInterest,
-        thinkingBehavioursFields.thinkingBehavioursAttitudesEmotionalIntimacy,
-        thinkingBehavioursFields.isUserSubmitted(stepUrls.sexualOffending),
+        thinkingBehavioursFields.thinkingBehavioursAttitudesRiskSexualHarm,
+        thinkingBehavioursFields.isUserSubmitted(stepUrls.riskOfSexualHarm),
         thinkingBehavioursFields.sectionComplete(),
       ].flat(),
-      next: stepUrls.thinkingBehaviours,
+      next: [
+        nextWhen(
+          thinkingBehavioursFields.thinkingBehavioursAttitudesRiskSexualHarm,
+          'YES',
+          stepUrls.riskOfSexualHarmDetails,
+        ),
+        stepUrls.summary,
```

Comment:
I've added this, looks like we require a default case for the routing logic in order to correctly calculated a resume URL